lib/Mojo/Base.pm

```diff
@@ -15,6 +15,10 @@ use IO::Handle ();
 my $NAME
   = eval { require Sub::Util; Sub::Util->can('set_subname') } || sub { $_[1] };
 
+# Role support requires Role::Tiny 2.000001+
+use constant ROLES =>
+  !!(eval 'require Role::Tiny' && $Role::Tiny::VERSION >= 2.000001);
```

Comment:
why not ` eval 'use Role::Tiny 2.000001 (); 1'; `
like at https://github.com/kraih/mojo/blob/master/lib/Mojo/IOLoop/TLS.pm#L11 ?